CMakeLists.txt

```diff
@@ -106,7 +106,7 @@ if(ODBC)
     endif()
     string(REGEX MATCH "/[^/\\]+$/gm" ODBC_LIB_NAME "${ODBC_LIBRARIES}") # regexr.com/698ls strips the preceding path from the actual lib name
     if (GMP_LIB_NAME)
-      add_compile_definitions(ODBC_DLNAME=${ODBC_LIB_NAME})
+      target_compile_definitions(GLPK PUBLIC ODBC_DLNAME=${ODBC_LIB_NAME})
```

Comment:
dumb question: why not using ODBC::ODBC

src: https://cmake.org/cmake/help/latest/module/FindODBC.html (since 3.12)

Reply:
That's actually a much better idea

Reply:
done